tests/kernel/mem_protect/mem_protect/src/mem_protect.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <kernel_structs.h>
+#include <string.h>
+#include <stdlib.h>
+
+/* Flag needed to figure out if the fault was expected or not. */
+extern bool valid_fault;
+
+/* For inherit.c */
+#define INHERIT_STACK_SIZE CONFIG_MAIN_STACK_SIZE
```

Comment:
if these defines are only used by inherit.c, just put them in inherit.c and not here

Reply:
Andrew I would like to have all the declarations in one place. In doing so I can change it in one location when needed. Hence kept it unchanged.